core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -276,6 +276,9 @@ class SparkContext(config: SparkConf) extends Logging {
 
   private[spark] def ui: Option[SparkUI] = _ui
 
+  /**
+   * @return `Option` containing URL to web UI.
```

Comment:
You said:
> url -> URL and ui -> UI, but, I don't this comment adds much to the reader beyond what the signature says already. I'd improve or remove it.

I thought you were talking about capitalizing as an improvement. I will remove these changes.